khmer/kfile.py

```diff
@@ -14,7 +14,7 @@
 from stat import S_ISBLK, S_ISFIFO
 
 
-def check_file_status(file_path, force):
+def check_file_status(file_path, force=False):
```

Comment:
I disagree. We left this out on purpose so that the scripts using this function will have to implement the force flag :-)